### PR TITLE
{bio}[foss/2017b] VEP v94.5 w/ Perl 5.26.0

### PR DESCRIPTION
**Pull request by vanzod**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by vanzod** [comment]
Test report by @vanzod
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
cermis - Linux debian 9.6, Intel(R) Xeon(R) CPU           E5620  @ 2.40GHz, Python 2.7.13
See https://gist.github.com/a23502bdc641d3dbc7caa3d27e016a73 for a full test report.

File: easybuild/easyconfigs/m/MariaDB/MariaDB-10.2.11-foss-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    '8295837e623f6c782e1d64b00e0877ea98cce4bf8846755bb86c8a7732797c19',  # MariaDB-10.1.13-link-rt-for-jemalloc.patch
+]
+
+dependencies = [
+    ('zlib', '1.2.11'),
+    ('ncurses', '6.0'),
+    ('jemalloc', '5.0.1'),
+    ('XZ', '5.2.3'),
+    ('Boost', '1.65.1'),
+    ('libxml2', '2.9.7'),
```
Comment (boegel) [inline easybuild/easyconfigs/m/MariaDB/MariaDB-10.2.11-foss-2017b.eb]: @vanzod Looks like this will need to be `2.9.4` to avoid introducing conflicts...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2447.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/a8adf45f7bfb0799a95dbc590f19ec94 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node2001.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/990ae6743b30170e35ea35789fd1849b for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in this PR)
node3100.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/883dee5fe198101bdda47ceb44a777c7 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @vanzod!

**Pull request merged**